src/components/NewMedium.vue

```diff
@@ -260,6 +217,23 @@ export default Vue.extend({
         })
         .then(() => (this.isLoading = false));
     },
+    postCompounds(mediumId) {
+      return this.compounds
+        .filter(({ id }) => id)
+        .map(compound => {
+          const payload = {
+            compound_identifier: compound.id,
+            compound_name: compound.name,
+            compound_namespace: "Metanetx",
```

Comment:
This is not a validi miriam identifier - should probably use this one https://www.ebi.ac.uk/miriam/main/collections/MIR:00000567 (also applies to everywhere else this namespace is referred to in this pr)